.github/workflows/bandit.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest]
```

Comment:
not needed anymore - also we lots windows build here - not sure if is by desing

Reply:
Needed, needed. Fixed.